core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -96,7 +96,9 @@ object ApiVersion {
     // Flexible version support in inter-broker APIs
     KAFKA_2_4_IV1,
     // No new APIs, equivalent to 2.4-IV1
-    KAFKA_2_5_IV0
+    KAFKA_2_5_IV0,
+    // Introduced StopReplicaRequest V3 containing the leader epoch for each partition (KIP-570)
+    KAFKA_2_6_IV0
```

Comment:
👀 I have added a new IBP version to gate the new API version. I was wondering if the was really necessary as the new version of the API is backward compatible.

Reply:
Yes, we bump the IBP every time one of the inter- broker protocol versions gets bumped. Hopefully soon brokers will use ApiVersions instead.